### DISCUSSION TITLE
Data Importer - how to import specific column for bank transaction fees

### DISCUSSION THREAD

**Discussion by karibuTW** [firefly-iii/firefly-iii#9121] [Firefly III] [answered]
Hello,
In Revolut, they add a transaction fee column in their export and I am not sure how to automate the import of this one with the rest.
In the file, I have:
- Transaction date
- Description
- Amount
- Fees

I can import all except the fees. Any idea how to import it and tag is automatically with bank fees transaction?
May be to import it under Amount debit or something like this?

Thank you

**Comment by JC5** [comment, marked as answer]
Mmmm, there is no way to import it yet, unless you run the import a second time and pretend the "fees" are the amount. But that would not merge them. There is a ticket for this, but it's not yet implemented.

**Reply by karibuTW** [reply]
Thanks JC5 for the quick reply.
Very clear. I do manually for now and create a transaction under bank fees, which is okay.
